website/docs/r/token.html.markdown

```diff
@@ -0,0 +1,65 @@
+---
+layout: "rancher2"
+page_title: "Rancher2: rancher2_token"
+sidebar_current: "docs-rancher2-resource-token"
+description: |-
+  Provides a Rancher v2 Token resource. This can be used to create tokens for Rancher v2 provider user and retrieve their information.
+---
+
+# rancher2\_token
+
+Provides a Rancher v2 Token resource. This can be used to create Tokens for Rancher v2 provider user and retrieve their information.
+
+There are 2 kind of tokens:
+- no scoped: valid for global system.
+- scoped: valid for just a specific cluster (`cluster_id` should be provided).
+
+Tokens can't be updated once created. Any diff in token data will recreate the token. If any token expire, Rancher2 provider will generate a diff to regenerate it.
+
+## Example Usage
+
+```hcl
+# Create a new rancher2 Token
+resource "rancher2_token" "foo" {
+  description = "foo token"
+  ttl = 1200
+}
+# Create a new rancher2 Token scoped
+resource "rancher2_token" "foo" {
+  cluster_id = "<cluster-id>"
+  description = "foo token"
+  ttl = 1200
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `cluster_id` - (Optional/ForceNew) Cluster ID for scoped token (string)
+* `description` - (Optional/ForceNew) Token description (string)
+* `enabled` - (Optional/ForceNew) Token enabled. Default `true` (bool)
```

Comment:
what does this do?

Reply:
`enabled` attribute is used control if the token is enabled. When you run `terraform apply` and the token is not enabled, the diff is detected and terraform will renew the token.